src/main/resources/run_protege_exporter.pl

```diff
@@ -1,11 +1,28 @@
 #!/usr/bin/perl
-
-use GKB::WebUtils;
 use strict;
 use warnings;
 
-my $wu = GKB::WebUtils->new_from_cgi();
-my $id = $ARGV[0] || die "No ID specified specified.\n";
-my $file_name = $ARGV[1] || die "No filename specified specified.\n";
+use CGI;
+
+use GKB::DBAdaptor;
+use GKB::WebUtils;
+
+my $cgi = CGI->new();
+
+my $user = $cgi->param("db_user");
+my $pass = $cgi->param("db_pass");
+my $host = $cgi->param("db_host");
+my $db = $cgi->param("DB");
+
+my $dba = GKB::DBAdaptor->new(
+    -user   => $user,
+    -pass   => $pass,
+    -host   => $host,
+    -dbName => $db
+);
+
+my $wu = GKB::WebUtils->new(-cgi => $cgi, -dba => $dba);
```

Comment:
Maybe this is a good time to change this variable name? If this is named $wu after Guanming -- it's not always his person instance this is returned, is it?

Reply:
Wait nevermind, this is for **W**eb**U**tils, so disregard my prior comment